core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/StackV4Endpoint.java

```diff
@@ -430,4 +431,11 @@ CertificatesRotationV4Response rotateAutoTlsCertificates(@PathParam("workspaceId
             notes = Notes.RENEW_CERTIFICATE_NOTES, nickname = "renewStackCertificate")
     FlowIdentifier renewCertificate(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name,
             @QueryParam("initiatorUserCrn") String initiatorUserCrn);
+
+    @PUT
+    @Path("internal/{name}/update_load_balancers")
```

Comment:
Does this also update loadbalancers that we use for datalake ha?  If not maybe rename it to "update_public_load_balaner"

Reply:
And corresponding new methods and classes.

Reply:
Yes, it will update all load balancers depending on what entitlements you have. I deliberately designed it to be broader than just the public LB, so that if we ever had need of that functionality we wouldn't have to re-code it. I realize the title is misleading in that regard.